bridge/opentracing/bridge_test.go

```diff
@@ -546,3 +546,49 @@ func TestBridge_SpanContext_IsSampled(t *testing.T) {
 		})
 	}
 }
+
+func TestBridge_SpanContext_Promoted_Methods(t *testing.T) {
+	bridge := NewBridgeTracer()
+	bridge.SetTextMapPropagator(new(testTextMapPropagator))
+
+	tmc := newTextCarrier()
+
+	type spanContextProvider interface {
+		HasTraceID() bool
+		TraceID() trace.TraceID
+		HasSpanID() bool
+		SpanID() trace.SpanID
+	}
+
+	testCases := []struct {
+		name    string
+		traceID trace.TraceID
+		spanID  trace.SpanID
+	}{
+		{
+			name:    "support for getting spanID and traceID",
+			traceID: [16]byte{byte(1)},
+			spanID:  [8]byte{byte(2)},
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
This is a lot of boilerplate to run a single test. Instead, can you just run the test directly?